src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -252,9 +253,9 @@ public void validate() throws AppEngineException {
      * The home directory of Google Cloud SDK. If not set, will attempt to look for the SDK in known
      * install locations.
      */
-    public Builder sdkPath(File sdkPathFile) {
+    public Builder sdkPath(Path sdkPathFile) {
```

Comment:
The comment seems questionable or perhaps misplaced. This method doesn't look anywhere.